### PR TITLE
Changes hit alerts from you to WE 

### PR DESCRIPTION
**Pull request by InsaneRed**
# About the pull request
This makes you are hit by x to We for the xeno side

i dont know if this is a good way to do this, however.
# Explain why it's good for the game

Consistency is good, also more hiveminded alerts

# Testing Photographs and Procedure
text based

# Changelog

:cl:
spellcheck: Changed hit alerts to We instead of You on the xeno side.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/powers/human_powers.dm [hunk 1]
```diff
@@ -202,7 +202,7 @@
 	if(new_resting == resting)
 		return
 	if(!COOLDOWN_FINISHED(src, rest_cooldown))
-		to_chat(src, SPAN_WARNING("You can't 'rest' that fast. Take a breather!"))
+		to_chat(src, SPAN_WARNING("[ishuman(src) ? "You" : "We"] can't 'rest' that fast. Take a breather!"))
```
Comment (ihatethisengine) [inline code/modules/mob/living/carbon/human/powers/human_powers.dm:205]:
```suggestion
		to_chat(src, SPAN_WARNING("[isxeno(src) ? "We" : "You"] can't 'rest' that fast. Take a breather!"))
```
Makes way more sense this way

**Comment by InsaneRed** [comment]
changed to check for isxeno() instead

**Review by spookydonut — COMMENTED** [review]
These are player feedback messages addressed to the player not the xeno as an IC RP element

**Comment by InsaneRed** [comment]
im aware but extra flair does not take away from that

File: code/modules/projectiles/guns/flamer/flamer.dm [hunk 2]
```diff
@@ -632,7 +632,7 @@
 		if(FIRE_VARIANT_TYPE_B)
 			if(isxeno(M))
 				var/mob/living/carbon/xenomorph/X = M
-				X.armor_deflection?(variant_burn_msg=" You feel the flames weakening your exoskeleton!"):(variant_burn_msg=" You feel the flaming chemicals eating into your body!")
+				X.armor_deflection?(variant_burn_msg=" We feel the flames weakening our exoskeleton!"):(variant_burn_msg=" You feel the flaming chemicals eating into your body!")
```
Comment (fira) [inline code/modules/projectiles/guns/flamer/flamer.dm:635]: Both messages are for xenos right? you should change both

File: code/modules/mob/living/carbon/human/powers/human_powers.dm [hunk 3]
```diff
@@ -211,24 +211,24 @@
...
 			set_lying_down()
 	else
 		if(body_position == STANDING_UP)
 			if(!silent)
-				to_chat(src, SPAN_NOTICE("You will now try to remain standing up."))
+				to_chat(src, SPAN_NOTICE("[isxeno(src) ? "We" : "You"] will now try to remain standing up."))
 		else if(HAS_TRAIT(src, TRAIT_FLOORED) || (buckled && buckled.buckle_lying != NO_BUCKLE_LYING))
 			if(!silent)
-				to_chat(src, SPAN_NOTICE("You will now stand up as soon as you are able to."))
+				to_chat(src, SPAN_NOTICE("[isxeno(src) ? "We" : "You"] will now stand up as soon as you are able to."))
```
Comment (fira) [inline code/modules/mob/living/carbon/human/powers/human_powers.dm:228]: There's still a you here

File: code/modules/mob/living/carbon/human/powers/human_powers.dm [hunk 4]
```diff
@@ -211,24 +211,24 @@
 	if(new_resting)
 		if(body_position == LYING_DOWN)
 			if(!silent)
-				to_chat(src, SPAN_NOTICE("You will now try to stay lying down on the floor."))
+				to_chat(src, SPAN_NOTICE("[isxeno(src) ? "We" : "You"] will now try to stay lying down on the floor."))
 		else if(HAS_TRAIT(src, TRAIT_FORCED_STANDING) || (buckled && buckled.buckle_lying != NO_BUCKLE_LYING))
 			if(!silent)
-				to_chat(src, SPAN_NOTICE("You will now lay down as soon as you are able to."))
+				to_chat(src, SPAN_NOTICE("[isxeno(src) ? "We" : "You"] will now lay down as soon as [isxeno(src) ? "We" : "You"] are able to."))
```
Comment (fira) [inline code/modules/mob/living/carbon/human/powers/human_powers.dm:217]: don't capitalize 2nd one

**Review by fira — COMMENTED** [review]
honestly i think this is a bad idea to absolutely want it everywhere, it's gonna be awkward to maintain. but if you absolutely want to maybe we can make a macro to make it easier

File: code/modules/projectiles/guns/flamer/flamer.dm (hunk 2 above)
Comment (InsaneRed) [reply code/modules/projectiles/guns/flamer/flamer.dm:635]: i believe thats for humans

Comment (InsaneRed) [reply code/modules/projectiles/guns/flamer/flamer.dm:635]: after testing this just doesnt get use danywhere for some reason

**Review by Drulikar — APPROVED** [review]
If people change their mind on this they can just make a PR to revert. Code looks fine.

**Pull request merged**